MatrixSDK/Space/MXSpaceService.swift

```diff
@@ -37,6 +37,9 @@ extension MXSpaceServiceError: CustomNSError {
 
 // MARK: - MXSpaceService notification constants
 extension MXSpaceService {
+    /// Posted once the first graph as been built or loaded
+    public static let didInitialised = Notification.Name("MXSpaceServiceDidInitialised")
```

Comment:
Is it normal that is not used by the app?
I expected to see it in the EI [PR](https://github.com/vector-im/element-ios/pull/4961).

Reply:
It will be used for https://github.com/vector-im/element-ios/issues/4570